src/themes/uswitch/theme.json

```diff
@@ -3954,6 +3924,60 @@
         }
       },
       "variants": {
+        "borderBottomImageLeft": {
+          "wrapper": {
+            "pb": [74, 100],
+            "px": 0
+          },
+          "inner": {
+            "flexDirection": ["column", "column", "row"],
+            "alignItems": ["center", "center", "flex-start"],
+            "px": 0,
+            "borderBottom": [null, null, "1px solid #C4C4C4"]
```

Comment:
can we name the color and add it to the theme

Reply:
theme-ui is slightly annoying for border colors, to take a value from the theme you can't use the border shorthand, i.e. you have to add a separate `borderColor: 'myColorName'`

Reply:
Fair, it might be worth going further and defining the border style itself if that's possible, as this exact border definition is used several times in the theme.json file. I'll take a look

Reply:
So yeah that would have been nice but doesn't seem possible. Will just do the colour